vespajlib/src/main/java/com/yahoo/geo/DistanceParser.java

```diff
@@ -0,0 +1,58 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+package com.yahoo.geo;
+
+public class DistanceParser {
+    // according to wikipedia:
```

Comment:
Consider adding link if this is from a single page.

Reply:
it's at least 3 pages, and not obvious where on the page we would refer to.  actually easier for the reader to just google the concept they wonder about if needed.